ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.js

```diff
@@ -466,7 +480,11 @@ define([
 
     setBboxUsng: function() {
       if (this.get('locationType') === 'usng') {
-        var result = converter.USNGtoLL(this.get('usngbb'))
+        let result
+        try {
+          result = converter.USNGtoLL(this.get('usngbb'))
+        } catch (err) {}
```

Comment:
❓ Same as my previous point. Why catch the error if you're going to suppress it? This leads to a lot of find spent debugging for errors down the road if we're not careful.

Reply:
https://github.com/codice/usng.js/issues/21 needs to be resolved before I can remove this try catch